### PR TITLE
Add newer verison of bootstrap icon

### PR DESCRIPTION
**Pull request by theblobscp**
Just adds the newer icon. Tell me if I forgot to add something, which I probbably did

### CODE REVIEW

**Comment by Thomas-Boi** [comment]
This looks good @theblobscp. Thank you for updating the icon.

@amacado @Panquesito7 should we remove the plain icons? If we do, should someone make a plain version to replace them? The original doesn't look icon friendly

**Comment by theblobscp** [comment]
Thanks! I'll try to fix it asap

**Comment by theblobscp** [comment]
I did it but now it looks terrible

**Comment by Thomas-Boi** [comment]
> I did it but now it looks terrible

Unfortunately, adding a `viewBox` attribute is not enough. You'd have to use the software specified by the bot or scale the elements manually down to fit the `viewBox` using an SVG editor.

**Comment by theblobscp** [comment]
@Thomas-Boi Does this work? If not, what do I need to do? I resized it using the thing from the bot

File: icons/bootstrap/bootstrap-original.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
...SShxfGA3CB4BUE/b09Hc/NO3Qqd9D/k6Hgl/87PtXGBqApKlwuF+0gEep8AEgNUZ1de3ddddh06fdh7zg9S8SAxR/VeCTgEkSoWuV/yekQMTuogJiQ95oEb8QQGacXLt+03sLNfF7Yb8e+vs+NVaTxh6k0AAkQZSuh/NK8/rKtxJx3LBm1FhKqTT6rebsC3fc+a3zjjtx9jOIFz+9fxVgF6BGjCjcr9N+PgBYNlRqjFy7q3vnv80+4uj7e3t7FfJ9/qD42e+vMowAqsOwf7Usq0B/Iw73Y48asqtahvs2kBorNw5YPQ9dduWlnzxs+qH39/b2OogWv3fLj6F/lWAEUFlCL9Kq3tYz5I8W4dsNIou0s+5Af8/zt91861333nvvHuRH9/V+PcVfA2gAKoc+Wu09mx/Q/R9HuG+nhRQZuU3CWbej64MnPvvZ83/2+uuvDyIfzntPJ+vC91ob7PNT/FWEBqB6uAAs27ZUqPjj7EG5t/Xi6lYSAdgpAZFWuyDcDxyZ3bKru+t3Ny+68fFHH320D/5+vC58k9cP9vkp/ipDA1BZXPjfyVeWZZsv4pH086OSKiQZkRsdErlfIywBYale2OgTQh6QUvZI6exy4O7o7O7+w4pnfvHCNddcsx3RQtaF7xrSg8aAVBkagOowfGFv3rzhjZZjx9+nbCmkVMKb2UdKNapn+5dKqsEBZ4+E62Szg519+3p2rN+0fvNrr73WuWjRoq5csaCAAbP31o2CE8hjyF9DRvVFWIdYyM+Lr8+Z502nBeQjBBsjn8EnKYKi1MXubZvKmKIBUz7FXyMYAVQW72LWp+gyzdbjhcH1aADCRBqMBsLu31P4o4h6uPjqDS8K8Cbn9AjOoV+v517F/I0Svbce7P+TGlGvF+FoxhTiB6fStg1po5W48N9URk/TvX0wndSY0X7x1StRRsBUdjRTrFCjIgJvm6IfZYz2i6+e0UVv8vj1eO7DxgL0dXr6OqIeL8J6I+j97UBePWAScXBarmIGCskoo14uwIOB4Jd96pGovj4hhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGE1AP/D9zQkSwrq6iWAAAAAElFTkSuQmCC"/>
```
Comment (Panquesito7) [inline icons/bootstrap/bootstrap-original.svg:8]: This is where you shouldn't use images. The file should be in pure SVG format.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
You may need to make some changes to the icon itself. Also, remember to make the SVG pure SVG format, not using external images or so (that's why the `B` looks blurry).

**Comment by Thomas-Boi** [comment]
If you resize the SVG using the web app, just specify the width and height to be 128 in the box specified.

image

You might have to uncheck the `Maintain Aspect Ratio` box if the SVG is not a square. Any further changes will need an SVG editor

**Comment by theblobscp** [comment]
> If you resize the SVG using the web app, just specify the width and height to be 128 in the box specified.
> 
> image
> 
> You might have to uncheck the `Maintain Aspect Ratio` box if the SVG is not a square. Any further changes will need an SVG editor

Ok

**Comment by theblobscp** [comment]
I'm gonna redo this pr, I kinda messed it up

**Pull request closed without merging**